lib/src/data/model/response_model.dart

```diff
@@ -0,0 +1,21 @@
+import '../../../src/domain/entity/movie.dart';
+
+class ResponseModel {
+  late int page;
+  late int totalResults;
+  late int totalPages;
+  List<Movie> results = <Movie>[];
```

Comment:
why you define late var here? You can directly create a ResponseModel constructor like
```
ResponseModel({
     this.page,
     this.totalResults,
     this.totalPages,
     this.result
});
```

Reply:
done